src/ragger/utils/misc.py

```diff
@@ -17,29 +17,17 @@
 from pathlib import Path
 
 
-def app_path_from_app_name(app_dir: Path, app_name: str, device: str) -> Path:
```

Comment:
Removing app_path_from_app_name means that this change is not retrocompatible.
Keeping it would make it an opt-in change (at conftest change). Maybe we can just deprecate it for now so that it doesn't break CI?
Or we can choose to explicitly break thing so that they are updated (and we don't have to keep deprecated stuffs).
What do you think about it?

Reply:
Nice catch about the retrocompatible, I'll add it back